css-ui-4/Overview.bs

```diff
@@ -2166,6 +2166,129 @@ Document languages should specify
 for each <a>widget</a> that they define
 what needs to be preserved.
 
+The possible kinds of widgets are ''appearance/none'', ''button'', ''searchfield'', ''textfield'',
+''slider-horizontal'', ''checkbox'', ''radio'', ''listbox'', ''menulist'', ''menulist-button'',
+''textarea'', ''progress-bar'', ''meter''.
+
+<div algorithm="compute kind of widget">
+    UAs must run this at [=used value=] time: compute the kind of widget to use for an element, |element|:
+
+    <ol>
+        <!-- none always works -->
+        <li>
+            If |computedAppearance| is ''appearance/none'', then return ''appearance: none''.
```

Comment:
This step needs to be moved down so it's after the step that defines *computedAppearance*. (This was my mistake, I only noticed it now when reviewing this :) )